### PR TITLE
Krustlet Episode

### PR DESCRIPTION
**Pull request by jeremyjung**
Please use https://drive.google.com/file/d/1bIxUfNBjDAj6RfNTx9orFe-S7agLbgkq/view?usp=sharing for audio. Removed a little bit where the guest was clearing their throat.

### CODE REVIEW

File: _episodes/030-krustlet.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Audio Editing: [Jeremy Jung](https://www.softwaresessions.com)
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: Jeremy Jung 
+
+Hosts: Jeremy Jung
+
+Guests: [Taylor Thomas](https://twitter.com/_oftaylor)
```
Comment (jonhoo) [inline _episodes/030-krustlet.md:63]: I would probably link to the guest's twitter as a link further up in the description as well :)

Comment (jeremyjung) [reply _episodes/030-krustlet.md:63]: i added it to the guest name at the top, is that what you were thinking?

**Comment by jeremyjung** [comment]
Oops, the corrected audio is missing the theme song. I will send a fixed version Wednesday. Sorry about that!

**Comment by jeremyjung** [comment]
Here's the corrected audio link that plays the theme for a few seconds at the start: https://drive.google.com/file/d/1f1sY6loQGjuhEUU_nT3OL8dqxlQC15V7/view?usp=sharing

**Comment by jonhoo** [comment]
@jeremyjung Great! I've uploaded it. You'll have to update the file size + duration to match the new file, and then we're good to go.

**Review by jonhoo — APPROVED** [review]

**Comment by jonhoo** [comment]
Published!

**Pull request merged**